src/views/ForgotPasswordPageChange/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import {Link, Route} from 'react-router-dom';
+import LoginPage from '../LoginPage/index';
+
+var requirements = {
+  password: undefined,
+  confirmpassword: undefined,
+}
+
+class ForgotPasswordPageChange extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
LOL I can tell you've been coding in C for CS136! Javascript doesn't need a ; at the end of the line (but it doesn't hurt to keep them in so don't worry about it!)